pom.xml

```diff
@@ -44,6 +44,8 @@
 
     <nexus.snapshot.repository>https://artifacts.camunda.com/artifactory/zeebe-io-snapshots/</nexus.snapshot.repository>
     <nexus.release.repository>https://artifacts.camunda.com/artifactory/zeebe-io/</nexus.release.repository>
+    <central.sonatype.deployment.name>${project.groupId}:${project.artifactId}:${project
```

Comment:
Ref: https://github.com/camunda/camunda-release-parent?tab=readme-ov-file#how-to-review-and-publish-artifact-deployment